site/downloads.markdown

```diff
@@ -71,6 +71,18 @@ Alternatively, many operating systems provide GHC, cabal and Stack through their
 
 **Do not use the Haskell development tools provided by Arch, they are broken.** For more information see [[1]](https://dixonary.co.uk/blog/haskell/cabal-2020) [[2]](https://stackoverflow.com/questions/65643699/what-is-the-suggested-way-of-setting-up-haskell-on-archlinux/65644318#65644318).
 
+#### NixOS
+
+[](This installation method is owned by @maralorn)
+
+* [Documentation for Haskell development and packaging with nix](https://haskell4nix.readthedocs.io/nixpkgs-users-guide.html#how-to-create-a-development-environment)
```

Comment:
Should "Nix" be capitalized? It seems so, [according to Wikipedia](https://en.wikipedia.org/wiki/Nix_(package_manager)).

```suggestion
* [Documentation for Haskell development and packaging with Nix](https://haskell4nix.readthedocs.io/nixpkgs-users-guide.html#how-to-create-a-development-environment)
```